backend/inspirehep/migrator/tasks.py

```diff
@@ -504,11 +515,30 @@ def migrate_record_from_mirror(
         return record
 
 
-def wait_for_all_tasks(task):
+def wait_for_all_tasks(task, retry_count=0):
     if not task:
         return None
-    LOGGER.info(f"Waiting for task completion.", waiting_task_id=task)
-    next_task = AsyncResult(task).get()
+    LOGGER.info(
+        f"Waiting for task completion.", waiting_task_id=task, retry_count=retry_count
+    )
+    try:
+        next_task = AsyncResult(task).get()
+    except ResponseError:
+        if retry_count < MAX_RETRY_COUNT:
+            LOGGER.exception(
+                "Redis ResponseError Exception, retrying",
+                task=task,
```

Comment:
```suggestion
                waiting_task_id=task,
```
to be consistent with the rest of logging (`task` is the name of the task)

Reply:
:+1: